sunpy/net/dataretriever/sources/goes.py

```diff
@@ -62,7 +62,7 @@ def _get_url_for_timerange(self, timerange, **kwargs):
         ----------
         timerange: sunpy.time.TimeRange
             time range for which data is to be downloaded.
-        satellite_number : int
+        satellitenumber : int
```

Comment:
CamelCase is for classes. I removed the underscore because of the way attrs are converted to kwargs